egs/zeroth_korean/s5/local/chain/tuning/run_tdnn_opgru_1a.sh

```diff
@@ -0,0 +1,291 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+# First the options that are passed through to run_ivector_common.sh
+# (some of which are also used in this script directly).
+stage=0
+nj=30
+train_set=train_clean
+speed_perturb=true
+test_sets="test_clean"
+gmm=tri4        # this is the source gmm-dir that we'll use for alignments; it
+                 # should have alignments for the specified training data.
+nnet3_affix=       # affix for exp dirs, e.g. it was _cleaned in tedlium.
+
+# Options which are not passed through to run_ivector_common.sh
+affix=1a  #affix for TDNN directory e.g. "1a" or "1b", in case we change the configuration.
+common_egs_dir=
+
+# OPGRU/chain options
+train_stage=-10
+get_egs_stage=-10
+xent_regularize=0.1
+label_delay=5
+max_param_change=2.0
+
+# training chunk-options
+chunk_width=150
```

Comment:
it wouldn't hurt to copy the chunk_width from the TDNN recipe.

Reply:
now it uses chunk_width=140,100,160